stdlib/OpenBLAS_jll/test/runtests.jl

```diff
@@ -14,4 +14,17 @@ end
 
 @testset "OpenBLAS_jll" begin
     @test dlsym(OpenBLAS_jll.libopenblas_handle, @blasfunc(openblas_set_num_threads); throw_error=false) != nothing
+    @testset "deterministic mul!" begin
```

Comment:
I don't think this test should be here, this is meant to test only very basic functionalities of the jll, like opening the library and such.

Reply:
Oh, this was [a suggestion](https://github.com/JuliaLang/julia/pull/53079#issuecomment-1913250911) from @KristofferC

Reply:
reverted